CONTRIBUTORS.md

```diff
@@ -164,6 +164,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
     <td align="center"><a href="https://github.com/JayantGoel001"><img src="https://avatars0.githubusercontent.com/u/54479676?v=4" width="100px;" alt=""/><br /><sub><b>Jayant Goel</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=JayantGoel001" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/Gaurav6982"><img src="https://avatars0.githubusercontent.com/u/49642550?v=4" width="100px;" alt=""/><br /><sub><b>Gaurav</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=Gaurav6982" title="Code">💻</a></td>
     <td align="center"><a href="http://techsfortalk.me"><img src="https://avatars3.githubusercontent.com/u/64386187?v=4" width="100px;" alt=""/><br /><sub><b>Ashutosh_K_Sah</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=ashu-cybertron" title="Code">💻</a></td>
+    <td align="center"><a href="https://github.com/charlie219"><img src="https://avatars2.githubusercontent.com/u/43098511?v=4" width="100px;" alt=""/><br /><sub><b>Akash Kumar Bhagat</b></sub></a><br /><a href="https://github.com/TesseractCoding/NeoAlgo/commits?author=charlie219" title="Code">💻</a></td>
```

Comment:
Yo don't need to update ```CONTRIBUTORS.md``` manually, you will be added as contributor once your code is merged.

Reply:
okay then I m revert the changes done to `CONTRIBUTORS.md` and add sample Input and Output